atom/app/atom_main.cc

```diff
@@ -6,13 +6,18 @@
 
 #include <stdlib.h>
 
+#include <memory>
```

Comment:
There's a lot going on this commit. Maybe we should have a bit more context to review what's going on in this commit.

Reply:
I split it up into multiple commits for that reason